posts/2023-06-29-OutreachyInternshipExperience/index.qmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Our Journey as Outreachy Interns with Bioconductor"
+description: |
+  Bioconductor participated in the Outreachy Internship program for the May 2023 cohort. Join us as we share our enriching experience working on various projects at Bioconductor and delve into the world of open science.
+author:
+  - name: Sonali Kumari
+    url: https://github.com/sonali8434
+  - name: Daena Rys
+    url: https://github.com/Daenarys8
+  - name: Atrayee Samanta
+    url: https://github.com/atrayees
+date: 2023-06-29
+categories:
+  - open science
+---
+
+## Introduction
+
+The world of open source offers abundant opportunities for growth and impact.
+However, diving into it can be daunting for newcomers. Thankfully, initiatives
+exist to bridge the gap between aspiring contributors and open source
+communities.
+
+Outreachy is one such project providing internships to underrepresented
+individuals in the tech industry. We were fortunate to be selected as interns at
+Bioconductor through Outreachy, where we experienced the open source world of
+computational biology. Whether you're a biologist seeking information on our
+tools or an Outreachy applicant interested in Bioconductor, this blog post will
+provide valuable insights!
+
+## Outreachy
+
+[Outreachy](https://www.outreachy.org/) is a program offering a paid 12-week
+remote internship in open source and open science. It aims to empower
+individuals facing systemic bias and underrepresentation in tech. Outreachy
+interns collaborate with experienced mentors on projects spanning programming,
+user experience, documentation, design, data science, marketing, advocacy, and
+community events.
+
+![Outreachy](outreachy_homepage.png)
+
+The application process involves three stages, where applicants contribute to
+1-2 projects of their choice. Through Outreachy, we were selected to work with
+Bioconductor on our chosen projects.
+
+## Bioconductor: Empowering Open Science
+
+Three interns and three Bioconductor community projects are participating in
+the May 2023 <a href="outreachy.org">Outreachy</a> internship round. In this blog
```

Comment:
Can we put the links in R Markdown format? (I sometimes forget too.)

    [Outreachy](outreachy.org)